src/platform/imx8ulp/include/platform/lib/dai.h

```diff
@@ -0,0 +1,26 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright 2021 NXP
+ *
+ * Author: Zhang Peng <peng.zhang_8@nxp.com>
+ */
+
+#ifdef __SOF_LIB_DAI_H__
+
+#ifndef __PLATFORM_LIB_DAI_H__
+#define __PLATFORM_LIB_DAI_H__
+
+#define DMAMUX2_SAI5_RX_NUM 69
```

Comment:
Can you please add a comment for what are these for?
I see we we don't have any DAIs in the `dai.h` files for imx8mp or imx8.

Where and for what are these used?

Reply:
It's in imx8ulp platform, only used in imx8ULP platform.

Reply:
@TangleZ where are these used? What exactly are these? 
I would expect a small explanation.

Please add a comment something like this **WARN: I don't know the hardware**
```
/* DMAMUX_ settings for mapping DMA channels to DAIs */
```

Reply:
On a second thought, please move DMAMUX stuff in a separate patch and explain why are they needed.